### ISSUE TITLE
Konsole's default profile is read-only

### ISSUE THREAD

**Issue by ttuegel** [NixOS/nixpkgs#132715]
### Describe the bug

In a new Plasma installation, Konsole's default profile is read-only. Attempting to change the profile creates a new profile (fine), but the next time you launch Konsole, it reverts to the default profile.

### Steps To Reproduce
1. Start Konsole in a clean Plasma installation.
2. Change some profile setting, e.g. the color scheme.
3. Restart Konsole.

### Expected behavior
The default profile is changed.

### Additional context
I think it is likely that Konsole copies its initial default profile settings from the Nix store, so we end up with a read-only file in the user's home directory. We may need to patch Konsole to touch up the file's permissions. This could affect other applications using KConfig.

### Notify maintainers
@ttuegel 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.52, NixOS, 21.11.git.08749d02382 (Porcupine)`
 - multi-user?: `no`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.14`
 - nixpkgs: `/etc/nixos/nixpkgs`

```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by hqurve** [comment]
Did you by chance change the default profile in settings -> manage profiles? Select the profile you want and say "Set as default"
![Screenshot from 2021-08-04 21-44-20](https://user-images.githubusercontent.com/53281855/128277959-a5b99b6a-6cd8-4349-bf89-46a2d9a48f03.png)

PS: while testing this I had a few issues (I think its how kde programs manage settings and I have similar problems with kde programs). My steps to reliably change any settings (including krunner's) is to change the settings and then close the application and reopen. Just opening a new instance upon changing the settings never seems to work for me

**Comment by ttuegel** [comment]
> Did you by chance change the default profile in settings -> manage profiles? Select the profile you want and say "Set as default"

Yes, that's a valid work-around, but as you can see in that screenshot, "Default" is marked read-only and it should not be.

>  My steps to reliably change any settings (including krunner's) is to change the settings and then close the application and reopen. Just opening a new instance upon changing the settings never seems to work for me

Yes, I did that.

**Comment by hqurve** [comment]
> "Default" is marked read-only and it should not be
I believe that "Default" should be read only as it is the template (however, I believe that the naming is bad)

I also went through other distros in VMs to see how they handle it. This is what was observed
- Debian distributions (debian 10 and kubuntu 20.10) both didnt have the "Default" profile listed (there was actually none listed)
- Arch, Fedora (with gnome) and tumbleweed all had a read-only "Default" profile.
